ghga_service_commons/api/mock_router.py

```diff
@@ -98,21 +109,34 @@ class MockRouter:
 
     def __init__(
         self,
-        http_exception_handler: Optional[
-            Callable[[httpx.Request, HttpException], Any]
-        ] = None,
+        exception_handler: Optional[Callable[[httpx.Request, E], Any]] = None,
+        exceptions_to_catch: Optional[tuple[Type[Exception], ...]] = None,
```

Comment:
Might be more intuitive to change the name slightly to `exceptions_to_handle`

Reply:
Agreed, changed in [620dcf9](https://github.com/ghga-de/ghga-service-commons/pull/19/commits/620dcf9b1d0f1f977b4db3885e7bbb13687525c4)